TombLib/TombLib/Catalogs/TEN Node Catalogs/UI_Hud.lua

```diff
@@ -0,0 +1,563 @@
+--User interface Nodes by Adngel,DaviDMRR and TrainWreck ; Code references by Lwmte and TEN Nodes.
+local CustomBar = require("Engine.CustomBar")
+
+-- !Name "Create basic custom bar"
+-- !Section "User interface"
+-- !Conditional "False"
+-- !Description "Creates a bar with max value 1000"
+-- !Arguments "NewLine, String, 50, [ NoMultiline ], Bar name"
+-- !Arguments "Number, [ 0 | 65535 | 2 ], {0}, 25, Start value of bar"

```

Comment:
There is no such type of argument as `Number`. It is `Numerical`. You don't see an error because TE silently defaults to a `Numerical` type.